iConverter/Pages/Main/Presenter/MainPresenter.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  MainPresenter.swift
+//  iConverter
+//
+//  Created by Aram Semerjyan on 13.04.22.
+//
+
+import Foundation
+import RxRelay
+
+final class MainPresenter: BasePresenter {
+    weak var vc: MainViewController?
+
+    private let router: MainRouter!
+
+    init(router: MainRouter) {
+        self.router = router
+
+        super.init()
+    }
+
+    func obserHistory(_ history: BehaviorRelay<[Transaction]>) {
```

Comment:
More appropriate would be:
```
showTransactions(_ transactions: [Transaction]) {
    vc?.viewModel.transactionsHistory.accept(transactions)
}
```

so you should remove bind functions in `MainViewModel`
then in MainInteractor

```
func loadHistory() {
        presenter.showTransactions(historyDataStore.loadTransactions())
   }
```